docs/content/reference/migration/migration-0-18.md

```diff
@@ -89,6 +90,18 @@ Note that the order of the method calls does _not_ affect the order in which tra
 an empty archetype instead that you can populate (e.g. `rerun::Transform3D().with_mat3x3(rerun::datatypes::Mat3x3::IDENTITY)`).
 
 
+Scale is no longer an enum datatype but a component with a 3D vec:
+Before:
+```rust
+let scale_uniform = rerun::Scale3D::Uniform(2.0);
+let scale_y = rerun::Scale3D::ThreeD([1.0, 2.0, 1.0]);
```

Comment:
We could keep `Uniform` and `ThreeD` around as functions, with `#[deprecated]` and `#[allow(non_snake_case_identifiers)]` to help the user